docs/datascience/jupyter-notebooks.md

```diff
@@ -21,53 +21,51 @@ MetaSocialImage: images/tutorial/social.png
 
 ## Setting up your environment
 
-To work with Jupyter notebooks, you must activate an Anaconda environment in VS Code, or another Python environment in which you've installed the [Jupyter package](https://pypi.org/project/jupyter/). To select an environment, use the **Python: Select Interpreter** command from the Command Palette (`kb(workbench.action.showCommands)`).
+To work with Python in Jupyter notebooks, you must activate an Anaconda environment in VS Code, or another Python environment in which you've installed the [Jupyter package](https://pypi.org/project/jupyter/). To select an environment, use the **Python: Select Interpreter** command from the Command Palette (`kb(workbench.action.showCommands)`).
```

Comment:
Depends. We need jupyter if the user is on a Mac M1 or an ubuntu machine that ZMQ binaries aren't available for.

Reply:
Hmm would it be worth it to add a note for Ubuntu and Mac M1 users? Or maybe we can detect which system they're on and show install Jupyter package depending on what machine they're on? @gregvanl is that possible?

Reply:
It would have to be pretty sophisticated. Mac M1 only. Mac older processors work fine. Ubuntu with specific glib libraries missing.

Reply:
Ah okay so it's just simpler to have everyone install the Jupyter package

Reply:
> Hmm would it be worth it to add a note for Ubuntu and Mac M1 users? Or maybe we can detect which system

I think we should just build the zmq binaries instead of documenting work arounds.
For instance we know what OS & linux distros VS Code supporst, so why not setup a CI to build zmq binaries for those particular OS/linux distros, its basically a one time job

Reply:
> Ah okay so it's just simpler to have everyone install the Jupyter package

I'd say its much simpler for us to build zmq & ship that, than having to deal with the issues users have when installing Jupyter & other packages (confusions around which interpreter/where it is installed, issues with users running `pip install` in conda environments , etc).

Reply:
I think the easiest thing for the docs to say now is that you install jupyter. Instead of having complicated logic for how the user determines what to install. We can change the docs when we fix the implementation problem.

Reply:
> We can change the docs when we fix the implementation problem.

Oh yes, I agree